pages/game.jsx

```diff
@@ -27,6 +29,9 @@ function game() {
           <UserControls />
           <UserControls />
         </div>
+        {started ? (<button className="bg-[#46CDD2] text-white my-5 py-2 m-auto flex justify-center items-center px-4 text-xl rounded-full hover:bg-[#4De1e7] hover:ease-in-out hover:duration-500 ease-out duration-500" onClick={(e) => {e.preventDefault(); setStarted(false); console.log("Stop battle!")}}>Stop!</button>) 
+        : (<button className="bg-[#46CDD2] text-white my-5 py-2 m-auto flex justify-center items-center px-4 text-xl rounded-full hover:bg-[#4De1e7] hover:ease-in-out hover:duration-500 ease-out duration-500" onClick={(e) => {e.preventDefault(); setStarted(true); console.log("Battle start!")}}>Battle!</button>)}
```

Comment:
Great!! Same comment as the other pull request. If you can declare the event handler function separately and call it inside the "onClick" attribute, it can be helpful for the long term readibility of the code. Other than that, great ✨